charts/agent/templates/_deployment-cluster-events-config.tpl

```diff
@@ -161,33 +160,24 @@ processors:
           - body["kind"] == "Deployment"
         statements:
           - set(attributes["observe_transform"]["facets"]["selector"], body["spec"]["selector"]["matchLabels"])
-          - set(attributes["observe_transform"]["facets"]["revision"], body["metadata"]["annotations"]["deployment.kubernetes.io/revision"])
-          - set(attributes["observe_transform"]["facets"]["desired"], body["spec"]["replicas"])
-          - set(attributes["observe_transform"]["facets"]["updated"], body["status"]["updatedReplicas"])
-          - set(attributes["observe_transform"]["facets"]["total"], body["status"]["replicas"])
-          - set(attributes["observe_transform"]["facets"]["available"], body["status"]["availableReplicas"])
-          - set(attributes["observe_transform"]["facets"]["unavailable"], body["status"]["unavailableReplicas"])
-          - set(attributes["observe_transform"]["facets"]["ready"], body["status"]["readyReplicas"])
-          - set(attributes["observe_transform"]["facets"]["ready"], 0) where attributes["observe_transform"]["facets"]["ready"] == nil
+          # - set(attributes["observe_transform"]["facets"]["revision"], body["metadata"]["annotations"]["deployment.kubernetes.io/revision"])
```

Comment:
Is this commented out on purpose? We can just remove it if it's not necessary

Reply:
Yeah let's remove it. Sorry!
(I was a little torn when thinking about this facet...So ended up in an inconsistent spot...Sorry!